### PR TITLE
feat(ui5-menu-item): add new `accessibleName` property

### PR DESCRIPTION
**Pull request by unazko**
The ui5-menu-items can now have an additional description provided
by the application developer.

Fixes: #5514
Fixes: #5656

### CODE REVIEW

**Review by tsanislavgatev — APPROVED** [review]

**Comment by tsanislavgatev** [comment]
Can we please add a short test for this, as we discussed ?

File: packages/main/src/MenuItem.js [hunk 1]
```diff
@@ -56,7 +56,19 @@ const metadata = {
 		},
 
 		/**
-		 * Indicates if the any of the element siblings have children items.
```
Comment (DavidMitreski5) [inline packages/main/src/MenuItem.js:59]: Indicates whether any of the element siblings have children items.

File: packages/main/src/MenuItem.js [hunk 2]
```diff
@@ -56,7 +56,19 @@ const metadata = {
...
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.6.0
+		 */
+		 accessibleName: {
+			type: String,
+		},
+
+		/**
+		 * Indicates if any of the element siblings have children items.
```
Comment (DavidMitreski5) [inline packages/main/src/MenuItem.js:71]: Indicates whether any of the element siblings have children items.

File: packages/main/src/MenuItem.js [hunk 3]
```diff
@@ -66,7 +78,7 @@ const metadata = {
 		},
 
 		/**
-		 * Indicates if the any of the element siblings have icon.
+		 * Indicates if any of the element siblings have icon.
```
Comment (DavidMitreski5) [inline packages/main/src/MenuItem.js:81]: Indicates whether any of the element siblings have an icon.

File: packages/main/src/MenuItem.js [hunk 4]
```diff
@@ -56,7 +56,19 @@ const metadata = {
 		},
 
 		/**
-		 * Indicates if the any of the element siblings have children items.
+		 * Defines the accessible ARIA name of the component.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.6.0
```
Comment (ilhan007) [inline packages/main/src/MenuItem.js:64]: 1.7.0

**Comment by ilhan007** [comment]
Could you check if this issue is also resolved https://github.com/SAP/ui5-webcomponents/issues/5656 seems similar to what is already referenced

File: packages/main/src/MenuItem.js (hunk 4 above)
Comment (ilhan007) [reply packages/main/src/MenuItem.js:64]: We are currently at 1.6.0, so this will go in 1.7.0

File: packages/main/src/MenuItem.js (hunk 3 above)
Comment (unazko) [reply packages/main/src/MenuItem.js]: Done

File: packages/main/src/MenuItem.js (hunk 2 above)
Comment (unazko) [reply packages/main/src/MenuItem.js]: Done

File: packages/main/src/MenuItem.js (hunk 4 above)
Comment (unazko) [reply packages/main/src/MenuItem.js]: Done

File: packages/main/src/MenuItem.js (hunk 1 above)
Comment (unazko) [reply packages/main/src/MenuItem.js:59]: Done

**Review by DavidMitreski5 — APPROVED** [review]

**Comment by ilhan007** [comment]
The feature is more for the MenuItem, so I will rename a bit the title, otherwise + 1

**Review by tsanislavgatev — APPROVED** [review]

**Pull request merged**